src/mlpack/core/data/load_csv.hpp

```diff
@@ -115,37 +204,75 @@ class LoadCSV
     inFile.clear();
     inFile.seekg(0, std::ios::beg);
     rows = 0;
-
+    
     while (std::getline(inFile, line))
     {
+      trim(line);
       ++rows;
```

Comment:
```suggestion
      ++rows;
       trim(line);
```

Reply:
Do you have any reason to move the trimming before moving to the next row?

Reply:
Nope, I keep it as it was before.